person.rb

```diff
@@ -1,15 +1,17 @@
 require_relative './nameable'
+require './rental'
 
 class Person < Nameable
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
   attr_reader :id
 
-  def initialize(age, name = 'Unknown', parent_permission: true)
+  def initialize(age, name = 'unknown', parent_permission: true)
```

Comment:
- [ ] Good job so far but you still have some issues that need to be fixed. When I try to create a student the console app failed `wrong number of arguments`. Please kindly fix that 🙃 🙃 

<img width="1168" alt="Screen Shot 2023-01-07 at 14 16 31" src="https://user-images.githubusercontent.com/74501523/211147472-cf899944-e97b-4836-9cf3-fdef33d26dcc.png">